modules/siddhi-extensions/event-table/pom.xml

```diff
@@ -76,23 +95,40 @@
 
     </dependencies>
 
+    <properties>
+        <org.eclipse.osgi.version>3.7.0.v20110613</org.eclipse.osgi.version>
+        <version.equinox.osgi.services>3.3.100.v20120522-1822</version.equinox.osgi.services>
+        <hazelcast.version>3.5.0.wso2v1</hazelcast.version>
+    </properties>
+
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.siddhi.extension.eventtable.hazelcast.internal.*
+                            org.wso2.siddhi.extension.eventtable.hazelcast.internal,
+                        </Private-Package>
                         <Export-Package>
+                            !org.wso2.siddhi.extension.eventtable.hazelcast.internal.*
+                            !org.wso2.siddhi.extension.eventtable.hazelcast.internal,
                             org.wso2.siddhi.extension.eventtable,
                             org.wso2.siddhi.extension.eventtable.*
                         </Export-Package>
                         <Import-Package>
                             org.apache.hadoop.util.bloom,
                             org.apache.hadoop.util.bloom.*,
+                            org.osgi.framework,
                             *;resolution:=optional
```

Comment:
Can you try to remove this resolution:=optional directive and include the needed dependencies with versions directly? It is recommended to provide the direct imports with versions whenever possible.